service/src/test/java/uk/gov/hmcts/reform/fpl/service/TaskListRendererTest.java

```diff
@@ -65,20 +59,9 @@
             task(SUBMIT_APPLICATION, NOT_AVAILABLE));
     }
 
-    @Test
-    void shouldRenderTaskListWhenApplicationDocumentsIsToggledOff() {
-        given(featureToggleService.isApplicationDocumentsEventEnabled()).willReturn(false);
-
-        final String expectedTaskList = readString("task-list/expected-task-list.md").trim();
-
-        assertThat(taskListRenderer.render(getTasks(DOCUMENTS))).isEqualTo(expectedTaskList);
-    }
-
     @Test
     void shouldRenderTaskListWhenApplicationDocumentsIsToggledOn() {
-        given(featureToggleService.isApplicationDocumentsEventEnabled()).willReturn(true);
-
-        final String expectedTaskList = readString("task-list/expected-task-list-when-application-documents-enabled.md")
+        final String expectedTaskList = readString("task-list/expected-task-list.md")
```

Comment:
APPLICATION_DOCUMENTS can be inlined, doesn't need to be parametric anymore since we don't have the toggle (getTasks(APPLICATION_DOCUMENTS))